data/config-schema/src/main/java/net/corda/schema/configuration/LedgerConfig.java

```diff
@@ -12,6 +12,8 @@ private LedgerConfig() {
     public static final String UTXO_TOKEN_SEND_WAKEUP_MAX_RETRY_DELAY = "tokens.sendWakeUpMaxRetryDelay";
     public static final String UTXO_TOKEN_CACHED_TOKEN_PAGE_SIZE = "tokens.cachedTokenPageSize";
     public static final String UTXO_TOKEN_CLAIM_TIMEOUT_SECONDS = "tokens.claimTimeoutSeconds";
-    public static final String UTXO_TOKEN_CACHE_REFRESH_PERIOD_MILLISECONDS = "tokens.tokenCacheExpiryPeriodMilliseconds";
+    public static final String UTXO_TOKEN_CACHE_EXPIRY_PERIOD_MILLISECONDS = "tokens.tokenCacheExpiryPeriodMilliseconds";
```

Comment:
Is the breaking change that renames "UTXO_TOKEN_CACHE_REFRESH_PERIOD_MILLISECONDS" to "UTXO_TOKEN_CACHE_EXPIRY_PERIOD_MILLISECONDS" worth it? The versioning number will have to be updated for this change. @vkolomeyko